otherlibs/Makefile.otherlibs.common

```diff
@@ -25,8 +25,8 @@ CAMLRUN ?= $(ROOTDIR)/boot/ocamlrun
 CAMLC := $(BEST_OCAMLC) -nostdlib -I $(ROOTDIR)/stdlib
 CAMLOPT := $(BEST_OCAMLOPT) -nostdlib -I $(ROOTDIR)/stdlib
 
-OC_CFLAGS += $(SHAREDLIB_CFLAGS) $(EXTRACFLAGS)
-OC_CPPFLAGS += -I$(ROOTDIR)/runtime
+OC_CFLAGS += $(SHAREDLIB_CFLAGS) $(EXTRACFLAGS) $(EXTRACPPFLAGS)
```

Comment:
why are we including the CPPFLAGS in the CFLAGS here? I would expect both OC_CFLAGS and OC_CPPFLAGS to be used by the compiler-caller down the line.

Reply:
`OC_CFLAGS` are the flags you pass to the C _compiler_ and should include everything you would pass to the C pre-processor (see also `configure.ac` and `runtime/Makefile`)

Reply:
I'm confused, `runtime/Makfile` has

```
define COMPILE_C_FILE
$(1).$(O): %.c
	$$(CC) -c $$(OC_CFLAGS) $$(OC_CPPFLAGS) $$(OUTPUTOBJ)$$@ $$<
endef
```

which suggests that PP-flags should *not* be included in `OC_CFLAGS` as they are already passed separately as `OC_CPPFLAGS`.

(So here I would maybe expect `OC_CPPFLAGS += EXTRACPPFLAGS`.)

Reply:
No, sorry, that's me being silly. You are correct - the `EXTRACPPFLAGS` should only be going into `OC_CPPFLAGS` as it's the rule in `Makefile.common` for compile C files which correctly includes `OC_CFLAGS` **and** `OC_CPPFLAGS`. It is however correct that the invocation for the depend files below does _not_ include `OC_CFLAGS` (and it was indeed the case beforehand - see https://github.com/ocaml/ocaml/pull/9332/files#diff-04a2085afcffb9fa32723e68b8957114L73)